src/parameterizations/vertical/MOM_CVMix_shear.F90

```diff
@@ -274,10 +283,10 @@ logical function CVMix_shear_init(Time, G, GV, US, param_file, diag, CS)
                  "Exponent of unitless factor of diffusivities, "// &
                  "for KPP internal shear mixing scheme." &
                  ,units="nondim", default=3.0)
-  call get_param(param_file, mdl, "SMOOTH_RI", CS%smooth_ri, &
```

Comment:
SMOOTH_RI should be formally obsoleted by adding a line to MOM_obsolete_params.F90, so that if a user tries to use the older parameter, they get a fatal error or warning (depending on whether they have set SMOOTH_RI = True or False) with a hint suggesting that they use N_SMOOTH_RI instead.  Our standard practice is that we will keep this warning around for a few years, and then we can delete it once we have some confidence that no users are likely to accidentally use the older parameter. We don't want people accidentally (and silently) running things that they do not intend to!